apache-rat-core/src/main/resources/org/apache/rat/asf.xml

```diff
@@ -33,9 +33,6 @@
 		<family id="UKOGL" name="UK Open Government License family" />
 		<family id="WTF" name="WTF Public License family" />
 
```

Comment:
@Claudenw do you remember what this file was for as it enables GPL by default, which is not allowed in ASF projects?

Reply:
Should we remove the file? I did not find any references to it from the creadur-site repo. WDYT?